vision/src/test/scala/com/lynbrookrobotics/potassium/vision/TargetTrackingTest.scala

```diff
@@ -0,0 +1,36 @@
+package com.lynbrookrobotics.potassium.vision
+
+import com.lynbrookrobotics.potassium.Signal
+import com.lynbrookrobotics.potassium.streams._
+import org.scalatest.FunSuite
+import squants.{Angle, Dimensionless, Length, Percent}
+import squants.space.{Degrees, Feet}
+import squants.time.Milliseconds
+
+class TargetTrackingTest extends FunSuite {
+  test("Cube with measured percent area 15.6 results in distance 2.75 feet away") {
+    val (percentArea, pubPercentArea) = Stream.manual[Option[Dimensionless]]
+    val camProps: Signal[VisionProperties] = Signal.constant(VisionProperties(Degrees(0), Feet(10.8645)))
+    val targetDistance: Option[Length] = Some(Feet(2.75))
+    val distanceToTarget = VisionTargetTracking.distanceToTarget(percentArea, camProps)
+    var lastTargetPosition: Option[Length] = Option(Feet(0))
+
+    distanceToTarget.foreach(lastTargetPosition = _)
+    pubPercentArea(Some(Percent(15.6)))
+
+    assert(targetDistance.get - lastTargetPosition.get < Feet(0.1))
```

Comment:
You should be setting an `implicit val tolerance = Feet(0.1)` and then doing `lastTargetPosition.get ~= Feet(2.75)` to make this test more clear.